source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1682,8 +1682,10 @@ export class CardElementPeer extends DesignerPeer {
             }
         }
 
+        let itemCount: number = 0;
```

Comment:
since we're immediately assigning this to `0`, we don't need an explicit type declaration